inspectit-oce-agent/src/system-test/resources/config/EntryPointDetectionTest.yml

```diff
@@ -0,0 +1,46 @@
+
+inspectit:
+  metrics:
+    definitions:
+      '[entrypoint/invocations]':
+        unit: invocations
+        views:
+          '[entrypoint/invocations]':
+            aggregation: COUNT
+            tags: {method_name : true}
+
+
+  instrumentation:
+    data:
+      entry_point_found:
+        is-tag: false
+      is_entry_point:
+        down-propagation: NONE
+        is-tag: false
+      method_name:
+        down-propagation: NONE
+
+    scopes:
+      EntryPointDetectionTest-methods:
+        type:
+          name: EntryPointDetectionTest
+          matcher-mode: ENDS_WITH
+        methods:
+          - name: methodA
+          - name: methodB
+
+    rules:
+      EntryPointDetectionTest-methods:
+        scopes:
+          EntryPointDetectionTest-methods: true
+        entry:
+          method_name: {provider: get_method_name}
+          entry_point_found: {provider: assign, constant-input: {val: true}}
+          is_entry_point:
+            only-if-null: entry_point_found
+            before:
+              entry_point_found : true
+            provider: assign
```

Comment:
Would it make sense to move the `assign` dp to the actual agent, thus, it is not only for testing existing?

Reply:
Then we should discuss the name, if assign is really the best choice. Other candidates are:
- copy
- set
- something else?